SharkCage/CageManager/CageManager.cpp

```diff
@@ -19,9 +20,29 @@
 #pragma comment(lib, "Rpcrt4.lib")
 
 NetworkManager network_manager(ContextType::MANAGER);
+std::optional<std::wstring> generateUuid();
```

Comment:
Pls change name to snake_case

Reply:
It’s a function, not a variable... those should be camelCase I suppose 🤓

Reply:
Ohhh you're right 👍 but then, the first letter should be in capital letters 😉

Reply:
Was thinking about capitalization as well, but we are not entirely consistent with this. Maybe we should start... 😅

Reply:
@SailReal but should it? I was under the inpresssion that `variables_are_snake`, `functionsAreLoverCamel` and `TypesAreUpperCamel` 🤷‍♂️😀

Reply:
I'm okay with leaving it as is, we can have a look at all function names further down the road